tracer_options.go

```diff
@@ -73,14 +73,21 @@ func (tracerOptions) PoolSpans(poolSpans bool) TracerOption {
 	}
 }
 
-// HostIPv4 creates a TracerOption that identifies the current service/process.
-// If not set, the factory method will obtain the current IP address.
+// HostIPv4 is deprecated, please use HostIP instead.
```

Comment:
Could you use the standard godoc for deprecation:
>To signal that an identifier should not be used, add a paragraph to its doc comment that begins with "Deprecated:" followed by some information about the deprecation. 

https://blog.golang.org/godoc-documenting-go-code